notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -117,7 +117,8 @@
    "metadata": {},
```

Comment:
<p>With the object-oriented proposal above, this code would be:</p><p> </p><pre class="ql-syntax" spellcheck="false">pipeline = ...
logreg_eucl_exp = Experiment(pipeline)
logreg_eucl_exp.run(cov_data)
logreg_eucl_exp.print_result()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>